frontend/javascripts/oxalis/controller/camera_controller.ts

```diff
@@ -241,6 +243,10 @@ export function rotate3DViewTo(id: OrthoView, animate: boolean = true): void {
     const paddingFactor = 1.1;
     width = Math.sqrt(datasetExtent.width ** 2 + datasetExtent.height ** 2) * paddingFactor;
     height = width / aspectRatio;
+    // Take the whole maximum diagonal width of the dataset to get the possible maximum extent of the dataset.
+    // This is used as an indication to set the far plane.
+    // A padding factor of 2 is used to ensure the adaptive far value is big enough in any case.
+    far = Math.sqrt(width ** 2 + datasetExtent.depth ** 2) * 2;
```

Comment:
That is the approximation I came up with. Does it make sense? The padding factor `1.1` does not seem sufficient here. Instead, something bigger is needed. I do not exactly know why `1.1` is not big enough and did not want to spend too much time thinking about this. Therefore I tried out 2 and it looks to be sufficient.

Reply:
> I do not exactly know why 1.1 is not big enough and did not want to spend too much time thinking about this.

Reading the code, I would assume this is due to the `clippingOffsetFactor` which is the maximum of width, height, and depth. The camera position is offset by that value in all 3 dimensions to make sure the whole dataset is visible. Therefore, a factor of 2 of the dataset diagonal makes sense to me. However, the code is new to me and I could have very well overlooked something.